JSON/GameModeOptions/GameModeOption_BowArrowOnly.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+	"$type": "GameModeLoader.Data.LevelOptionCatalog, GameModeLoader",
+	"id": "only_bow_arrow",
+	"version": 3,
+	"excludeLevelIds": [],
+	"excludeGameModeNames":  ["Survival"],
```

Comment:
Do you want to exclude it from survival gamemode? If you want to have only bow and arrow in Survival mode, you can remove `"Survival"`